e2e-tests/constants/examples/phone-number-examples.js

```diff
@@ -12,6 +12,7 @@ export const INVALID_PHONE_NUMBERS = {
   INTERNATIONAL: '0033144513100',
   INTERNATIONAL_PLUS: '+337906 208973',
   TOO_SHORT: '*44',
+  ABOVE_MAX_CHARS: '02072718010'.repeat(50),
```

Comment:
Do you think it'd be nicer if we did `'1'.repeat(192)`? the `repeat(50)` is a bit vague whereas having 192 shows what the maximum is?